src/bin/Interpret.cc

```diff
@@ -23,6 +23,8 @@ OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
 WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 *********************************************************************/
 
+#include <cstdio>
+#include <CounterRewriter.h>
```

Comment:
I guess these are related to the experimentation.  We probably shouldn't put them to the master.